openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -142,14 +142,20 @@
   narrative:
     - key: a
       text: |
-        A control response is planned. Engineering progress can be tracked via:
+        In case writing an audit record fails, all the API servers increase the
```

Comment:
Should we switch this to `complete`? admins can do this on their own, and the fact that we'll add this to the compliance-operator is just a plus really.

Reply:
Good point, I also improved the reply to b)